src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelMetadata.cs

```diff
@@ -308,6 +308,13 @@ public virtual ModelMetadata ContainerMetadata
         /// <value>Defaults to <c>null</c>.</value>
         public virtual IPropertyValidationFilter PropertyValidationFilter => null;
 
+        /// <summary>
+        /// Gets a predicate which determines whether or not the model should be bound based on state
+        /// from the current request.
+        /// </summary>
+        /// <value>Defaults to <c>null</c>.</value>
+        public virtual Func<ActionContext, bool> RequestPredicate => null;
```

Comment:
Will this addition be needed if `GetBindingInfo(ModelMetadata modelMetadata)` changes to `GetBindingInfo(ModelMetadata modelMetadata, BindingInfo initialBindingInfo)`?